.eslintrc.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
     '@metamask/eslint-config/config/nodejs',
   ],
 
+  rules: {
+    'prefer-object-spread': 'off',
```

Comment:
We should probably move `prefer-object-spread` out of the primary config in `eslint-config` 🤔 I'll create an issue there for it.

Reply:
https://github.com/MetaMask/eslint-config/issues/50